### PR TITLE
Dpp 1218 remove duplicate payments for programme type

### PR DESCRIPTION
**Pull request by carlbm**
There are some files re the integration tests that aren't needed, but I left them in anyway.
I've also left in the whitespace changes to make the SQL more readable

### CODE REVIEW

File: src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/10 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarningsRequiringPayments.sql [hunk 1]
```diff
@@ -47,10 +47,10 @@ JOIN Staging.CollectionPeriods cp
 LEFT JOIN Reference.RequiredPaymentsHistory ph
               ON ae.Ukprn = ph.Ukprn
               AND ae.LearnRefNumber = ph.LearnRefNumber
-              --AND ae.LearnRefNumber = ph.LearnRefNumber
```
Comment (tom-gough) [inline src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/10 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarningsRequiringPayments.sql:50]: commented out code can be removed

File: src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenMathsAndEnglishPaymentsAreCalculated.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public class WhenMathsAndEnglishPaymentsAreCalculated
+    {
+        [SetUp]
+        public void Arrange()
+        {
+            TestDataHelper.Clean();
+        }
+
+        [Test]
+        [Ignore("Harness does not support multi-period-end tests")]
```
Comment (tom-gough) [inline src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenMathsAndEnglishPaymentsAreCalculated.cs]: why add a test and immediately ignore it?

File: src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenLearnerOnAStandardHasADifferentProgType.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    public class WhenLearnerOnAStandardHasADifferentProgType
+    {
+        [SetUp]
+        public void Arrange()
+        {
+            TestDataHelper.Clean();
+        }
+
+        [Test]
+        [Ignore("Harness does not support multi-period-end tests")]
```
Comment (tom-gough) [inline src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenLearnerOnAStandardHasADifferentProgType.cs]: why add a test and immediately ignore it?

File: src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenLearnerOnAStandardHasADifferentProgType.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+            TestDataHelper.CopyReferenceData();
+
+            // Act
+            context = new ExternalContextStub();
+            task = new PaymentsDueTask();
+            task.Execute(context);
+
+            // TODO: Make these work somewhere close to reality
+            //TestDataHelper.CopyToDeds();
```
Comment (tom-gough) [inline src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Issues/WhenLearnerOnAStandardHasADifferentProgType.cs]: why is this commented out? what is the todo about?

**Review by tom-gough — CHANGES_REQUESTED** [review]

File: src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/10 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarningsRequiringPayments.sql (hunk 1 above)
Comment (tom-gough) [reply src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/10 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarningsRequiringPayments.sql:50]: thought this was an addition not a removal, never mind

**Review by tom-gough — APPROVED** [review]

**Comment by satya-sanikommu** [comment]
Functional testing is done on VMs and the fix has been verified on Shadow with R07 data by validating the results. This will be further tested on Pre-Prod for an End-To-End and on Shadow for validating results based on R08 data. Ready to merge

**Pull request merged**